src/docs/markdown/build.md

```diff
@@ -4,10 +4,24 @@ title: "Build from source"
 
 # Build from source
 
+There are 2 ways to build Caddy, as opposed to installation of pre-built executable:
+- Build from Git repo
+- Build using `xcaddy`
+
 Requirements:
 
 - [Go](https://golang.org/doc/install) 1.20 or newer
 
+You can easily cross-compile, i.e. build Caddy on one operating system or architecture for a different, using the same tooling. You'll find details on this in the respective section.
+
+The latest section contains instructions for users who installed Caddy using the APT command on Debian-derivative system yet need the custom build executable for their operations.
```

Comment:
"latest" usually feels like it relates to time rather than position in an article. "last" is usually better for that.

This should also probably link to the section below.